### ISSUE TITLE
Websocket: ws+meta.nchan protocol - no echo with empty message

### ISSUE THREAD

**Issue by iotch** [slact/nchan#313]
Hello!

When using `ws+meta.nchan` protocol on a pub-sub location nothing gets echoed back when message is empty string:
```js
var ws = new WebSocket('wss://...', 'ws+meta.nchan');
ws.onopen = function() {
    ws.send('');
}
ws.onmessage = function(e) {
    console.log(e.data); // message with metadata expected
}
ws.onclose = function(e) {
    console.log(e.code, e.reason);
}
```

Nginx logs error: `[alert] 26683#26683: *13 zero size buf in output t:0 r:0 f:0 0000000000000000 0000000000000000-0000000000000000 0000000000000000 0-0`

Technically, an empty string is a message too.

Thanks!

**Comment by slact** [comment]
Fixed in 1869d35d .

**Comment by iotch** [comment]
Thanks!
